### PR TITLE
Support for OCP-3.10 added

### PR DESCRIPTION
**Pull request by puppet-py**
- updated the supported flavors by acc-provision
- orders have been changed

### CODE REVIEW

File: provision/acc_provision/flavors.yaml [hunk 1]
```diff
@@ -108,6 +108,24 @@ flavors:
     status: null
     hidden: false
     order: 1
+  openshift-3.10:
+    desc: Red Hat OpenShift Container Platform 3.10
+    default_version: 1.9
```
Comment (snaiksat) [inline provision/acc_provision/flavors.yaml]: I know we have been setting this to 1.9 earlier, but is this correct for OpenShift?

**Comment by snaiksat** [comment]
@puppet-py please see my inline comment.

**Comment by snaiksat** [comment]
I will merge this for now. My inline comment can be discussed separately.

**Comment by snaiksat** [comment]
Oh it needs a rebase first!

**Comment by puppet-py** [comment]
Created a new branch and new PR, closing this

**Pull request closed without merging**